### ISSUE TITLE
Make opaBlastRadius configurable from Azure DevOps gui using variables

### ISSUE THREAD

**Issue by simongottschlag** [XenitAB/azure-devops-templates#36]
We need to be able to change the opaBlastRadius for terraform pipelines using variables in the AzDO gui.

**Comment by CalleB3** [comment]
This can be done by adding configuration:

```bash

parameters:
  - name: BlastRadius
    type: string
    default: "50"

variables:
  - name: opaBlastRadius
    value: ${{ parameters.BlastRadius }}

```

Since we dont/cant have centrally ajusted pipelines, like the templates repo for XKF. Keeping the issue open until this is done for all pipelines.

**Comment by simongottschlag** [comment]
@CalleB3 is it possible to use this in a PR?

Example: You make a change that causes the blast radius to go to 100. You want to pass the PR validation and adjust this to 101.

**Comment by CalleB3** [comment]
@simongottschlag Yes, you can. When PR validation fails, open the failed pipeline run and do "Run New" with the prefilled "PR merge branch" and set a new BlastRadius in the GUI. Such as:
![image](https://user-images.githubusercontent.com/91601778/230112345-5ca51d12-feec-4a09-beb4-b213eb6ef505.png)

But as always, there is the strange behavior of Azure DevOps sometime not recognising the "rerun" as part of the PR and then not showing the run as passed PR validation.
